drivers: sensors: AMS Air quality Sensor Implementations (#10431)
zephyrproject-rtos/zephyr: samples/sensor/ams_iAQcore/README.rst

.. _ams_iaqcore:

ams iAQcore Indoor air quality sensor
#####################################

Overview
********

This sample application demonstrates how to use the ams iAQcore sensor to
measure CO2 equivalent and VOC. The CO2 value is a predicted value derived from
VOC. The values are fetched and printed every second.

Building and Running
********************

This sample application uses the sensor connected to the i2c stated in the
iaq_core.overlay file.

> Flash the binary to a board of choice where the sensor is connected to.

Reviewer: ```suggestion
Flash the binary to a board of choice with a sensor connected.
```

.. zephyr-app-commands::
   :zephyr-app: samples/sensors/ams_iAQcore

>    :board: any with i2c

Author: Done.
